src/utils/uploader.js

```diff
@@ -112,18 +112,35 @@ export class BlobUploader {
         tx.blobVersionedHashes = versionedHashes;
         tx.blobs = ethersBlobs;
         tx.kzg = kzg;
-        return await this.#wallet.sendTransaction(tx);
+        return tx;
     }
 
-    async sendTxLock(tx, blobs) {
+    async sendTx(tx, blobs) {
+        const txObj = await this.#populate(tx, blobs);
+        return await this.#wallet.sendTransaction(txObj);
+    }
+
+    async #sendTxLock(tx) {
         const release = await this.#mutex.acquire();
         try {
-            return await this.sendTx(tx, blobs);
+            tx.nonce = await this.getNonce();
+            return await this.#wallet.provider.broadcastTransaction(await this.#wallet.signTransaction(tx));
```

Comment:
What is the difference between this line and wallet.sendTransaction?

Reply:
There are many network requests inside ‘sendTransaction’